README.md

```diff
@@ -0,0 +1,104 @@
+# setup-nomad-pack
+
+The `hashicorp/setup-nomad-pack` Action sets up the `nomad-pack` CLI in your GitHub Actions workflow by adding the binary to `PATH`.
+
+## Table of Contents
+
+- [setup-nomad-pack](#setup-nomad-pack)
+  - [Table of Contents](#table-of-contents)
+  - [Requirements](#requirements)
+  - [Usage](#usage)
+  - [Author Information](#author-information)
+  - [License](#license)
+
+## Requirements
+
+This GitHub Actions supports all commands that are available in the `nomad-pack` CLI.
+
+The `run`, `destroy`, `info`, and `status` commands require access to a Nomad cluster, as defined through the environment variable `NOMAD_ADDR` (and optionally: `NOMAD_TOKEN`).
+
+## Usage
+
+1.) Create GitHub Actions Secrets by going to the repository's _Settings_ tab, followed by expanding the _Secrets_ sidebar, and finally _Actions_.
+
+- Set the `NOMAD_ADDR` to the IP-address or hostname of a Nomad cluster that is routable for GitHub Actions Runners.
```

Comment:
Same meta comment.  There may need to be more secrets configured based on the user's environment.

Reply:
Great catch here, too! I've added a link to the docs.